start-tests.sh

```
@@ -358,163 +210,44 @@ if [[ "$PROW_MODE" == true ]]; then
-    else
-        echo "Proceeding with test setup"
-    fi
-  done
-fi
+echo "Waiting up to 10 minutes for search pods to reach Running status"
+SEARCH_RUNNING="false"
+ATTEMPTS=0
+MAX_ATTEMPTS=60
+INTERVAL=10
+while [[ "${SEARCH_RUNNING}" == "false" ]] && (( ATTEMPTS != MAX_ATTEMPTS )); do
+  RUNNING_SEARCH_PODS_COUNT=($(oc get pods --all-namespaces -l app=search --field-selector=status.phase==Running --no-headers | wc -l))
```

Shellcheck (0.11.0)

[warning] Prefer mapfile or read -a to split command output (or quote to avoid splitting).

(SC2207)